helm/tenant/templates/tenant.yaml

```diff
@@ -177,22 +177,24 @@ spec:
     {{- end }}
     ## Key name to be created on the KMS, default is "my-minio-key"
     keyName: {{ .kes.keyName | quote }}
-    {{- with (dig "resources" (dict) .) }}
-    resources: {{- toYaml . | nindent 4 }}
+    {{- with (dig "kes" "resources" (dict) .) }}
```

Comment:
I guess we need a fallback to the generic values to ensure backward compatibility (also for the other values).

Reply:
Just pushed a version which keeps bw compatibility.

Reply:
affinity values were not taken from the right place. Fixed.